static/src/javascripts/projects/common/modules/experiments/tests/viewability.js

```diff
@@ -17,16 +25,62 @@ define(function () {
             return true;
         };
 
+        var showMessage = function (panelLinks) {
+            var templateStr =
```

Comment:
I think that it will be better to use html template instead of string here. Look how I did this last time: https://github.com/guardian/frontend/pull/9334/files (invite-to-panel.html)